src/main/java/org/gridsuite/modification/server/repositories/ModificationRepository.java

```diff
@@ -32,6 +32,9 @@ public interface ModificationRepository extends JpaRepository<ModificationEntity
     @Query(value = "SELECT new ModificationEntity(m.id, m.type, m.date, m.stashed, m.messageType, m.messageValues) FROM ModificationEntity m WHERE m.group.id = ?1 order by m.modificationsOrder")
     List<ModificationEntity> findAllBaseByGroupId(UUID uuid);
 
+    @Query(value = "SELECT m FROM ModificationEntity m WHERE m.group.id = ?1 AND m.stashed = ?2 order by m.modificationsOrder")
+    List<ModificationEntity> findAllStashedByGroupId(UUID uuid, Boolean stashed);
+
```

Comment:
`@Query("SELECT m FROM ModificationEntity m WHERE m.group.id = :groupId AND m.stashed = :stashed ORDER BY m.modificationsOrder")
List<ModificationEntity> findAllStashedByGroupId(@Param("groupId") UUID groupId, @Param("stashed") Boolean stashed);
`